docs/reference/ccr/getting-started.asciidoc

```diff
@@ -253,6 +253,11 @@ PUT /server-metrics-copy/_ccr/follow?wait_for_active_shards=1
 
 //////////////////////////
 
+The follower index is bootstrapped using the <<remote-recovery, remote recovery>>
```

Comment:
I find "bootstrapped" unclear. Could we another verb like "initialized", "prepared", "loaded", ...?